saas/models.py

```diff
@@ -1528,6 +1528,26 @@ def provider(invoiced_items):
             result = get_current_provider()
         return result
 
+    def by_organization(self, organization, transaction_type):
```

Comment:
Okay, would you like me to make this change?

Regarding custom querysets, I still don't see the motivation to use them in this case. In general, I still see them only as a ways to bundle together several `QuerySet` API calls, e.g. `filter`, and to make them extensible. This is desirable if you have a bunch of related `QuerySets` across multiple model types, e.g. you have more than one model with a common set of fields. Then you can use inheritance for DRY.

Do you see another motivation for using custom querysets? And even if you don't, would you like me to change this to be one?